provisiondomainmanager_policy.tf

```diff
@@ -0,0 +1,527 @@
+      "rds:AddTagsToResource",
+      "rds:AuthorizeDBSecurityGroupIngress",
+      "rds:CopyDBClusterSnapshot",
+      "rds:CopyDBParameterGroup",
+      "rds:CopyDBSnapshot",
+      "rds:CopyOptionGroup",
+      "rds:CreateDBCluster",
+      "rds:CreateDBClusterEndpoint",
+      "rds:CreateDBClusterParameterGroup",
+      "rds:CreateDBClusterSnapshot",
+      "rds:CreateDBInstance",
+      "rds:CreateDBParameterGroup",
+      "rds:CreateDBSecurityGroup",
+      "rds:CreateDBSnapshot",
+      "rds:CreateDBSubnetGroup",
+      "rds:CreateEventSubscription",
+      "rds:CreateOptionGroup",
+      "rds:DeleteDBCluster",
+      "rds:DeleteDBClusterEndpoint",
+      "rds:DeleteDBClusterParameterGroup",
+      "rds:DeleteDBClusterSnapshot",
+      "rds:DeleteDBInstance",
+      "rds:DeleteDBInstanceAutomatedBackup",
+      "rds:DeleteDBParameterGroup",
+      "rds:DeleteDBSecurityGroup",
+      "rds:DeleteDBSnapshot",
+      "rds:DeleteDBSubnetGroup",
+      "rds:DeleteEventSubscription",
+      "rds:DeleteOptionGroup",
+      "rds:DescribeDBClusterEndpoints",
+      "rds:DescribeDBClusterParameterGroups",
+      "rds:DescribeDBClusterParameters",
+      "rds:DescribeDBClusterSnapshotAttributes",
+      "rds:DescribeDBClusterSnapshots",
+      "rds:DescribeDBClusters",
+      "rds:DescribeDBInstanceAutomatedBackups",
+      "rds:DescribeDBInstances",
+      "rds:DescribeDBParameterGroups",
+      "rds:DescribeDBParameters",
+      "rds:DescribeDBSecurityGroups",
+      "rds:DescribeDBSnapshotAttributes",
+      "rds:DescribeDBSnapshots",
+      "rds:DescribeDBSubnetGroups",
+      "rds:DescribeEngineDefaultClusterParameters",
+      "rds:DescribeEngineDefaultParameters",
+      "rds:DescribeEventCategories",
+      "rds:DescribeEventSubscriptions",
+      "rds:DescribeEvents",
+      "rds:DescribeOptionGroupOptions",
+      "rds:DescribeOptionGroups",
+      "rds:ListTagsForResource",
+      "rds:ModifyDBCluster",
+      "rds:ModifyDBClusterEndpoint",
+      "rds:ModifyDBClusterParameterGroup",
+      "rds:ModifyDBClusterSnapshotAttribute",
+      "rds:ModifyDBInstance",
+      "rds:ModifyDBParameterGroup",
+      "rds:ModifyDBSnapshot",
+      "rds:ModifyDBSnapshotAttribute",
+      "rds:ModifyDBSubnetGroup",
+      "rds:ModifyEventSubscription",
+      "rds:ModifyOptionGroup",
+      "rds:RebootDBInstance",
+      "rds:RemoveTagsFromResource",
+      "rds:ResetDBClusterParameterGroup",
+      "rds:ResetDBParameterGroup",
+      "rds:RestoreDBClusterFromS3",
+      "rds:RestoreDBClusterFromSnapshot",
+      "rds:RestoreDBClusterToPointInTime",
+      "rds:RestoreDBInstanceFromDBSnapshot",
+      "rds:RestoreDBInstanceFromS3",
+      "rds:RestoreDBInstanceToPointInTime",
+      "rds:RevokeDBSecurityGroupIngress",
+      "rds:StartDBCluster",
+      "rds:StartDBInstance",
+      "rds:StopDBCluster",
+      "rds:StopDBInstance"
+    ]
+
+    resources = [
+      "arn:aws:lambda::${local.userservices_account_id}:layer:domain-manager-*",
```

Comment:
If it were me, I would break things like this up into three statement blocks - one for each of `rds`, `logs`, and `lambda`.

Reply:
Excellent suggestion- implemented in https://github.com/cisagov/cool-accounts-userservices/pull/3/commits/5ccc0ec2729f22aa263a9077ca86236e1959e0c3.  Thanks!